root_pandas/readwrite.py

```diff
@@ -147,6 +148,18 @@ def do_flatten(arr):
         arr = append_fields(arr_, '__array_index', idx, usemask=False, asrecarray=True)
         return arr
 
+    def remove_high_dimensions(arr):
```

Comment:
I wanted to leave the name flexible. But I am okay calling it like this, in which case I will also skip the explicit `allowed_dimensions = [0]` constant.